code/game/gamemodes/miniantags/abduction/machinery/dispenser.dm

```diff
@@ -71,6 +71,9 @@
 			return TRUE
 
 /obj/machinery/abductor/gland_dispenser/attack_hand(mob/user)
+	if(!isabductor(user))
+		to_chat(user, "<span class='warning'>You don't understand any of the alien writing!</span>")
+		return
```

Comment:
Can you also add this to the ui act after if(..()) return?

Reply:
Done!